diff --git a/src/org/usfirst/frc/team5129/robot/Robot.java b/src/org/usfirst/frc/team5129/robot/Robot.java
index d6afcad..4031b6d 100644
--- a/src/org/usfirst/frc/team5129/robot/Robot.java
+++ b/src/org/usfirst/frc/team5129/robot/Robot.java
@@ -76,10 +76,12 @@ public class Robot extends IterativeRobot {
 
 	@Override
 	public void autonomousPeriodic() {
+		int choose = Integer.parseInt(choice.getTable().getString(
+				"autonomous_routine", "0"));
 		for (AutoSubsystem s : auto) {
 			s.tick();
 		}
-		switch (0) {
+		switch (choose) {
 			case 0:
 				auto[0].complete(3, routine);
 				break;
